src/RageTimer.cpp

```diff
@@ -29,13 +29,14 @@
 
 #include <cmath>
 #include <cstdint>
+#include <atomic>
 
 const std::uint64_t ONE_SECOND_IN_MICROSECONDS_ULL = 1000000ULL;
 const std::int64_t ONE_SECOND_IN_MICROSECONDS_LL = 1000000LL;
 const double ONE_SECOND_IN_MICROSECONDS_DBL = 1000000.0;
 
 const RageTimer RageZeroTimer(0,0);
-static std::uint64_t g_iStartTime = ArchHooks::GetMicrosecondsSinceStart( true );
+static std::atomic<std::uint64_t> g_iStartTime = ArchHooks::GetMicrosecondsSinceStart(true);
```

Comment:
Fwiw, atomic variables come at a performance cost and I doubt the usefulness of this PR given my previous comment.